packages/flutter/test/widgets/image_resolution_test.dart

```diff
@@ -18,27 +19,31 @@ import '../image_data.dart';
 ByteData testByteData(double scale) => ByteData(8)..setFloat64(0, scale);
 double scaleOf(ByteData data) => data.getFloat64(0);
 
-const String testManifest = '''
+final Map<dynamic, dynamic> testManifest = json.decode('''
```

Comment:
Here and elsewhere, its always safe to replace `dynamic` with `Object?`. We should prefer to use Object? to avoid accidentally including dynamic calls. Not super important for tests, but its still helpful to stay consistent